src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -143,4 +156,17 @@ public Preferences preferenceFactory(Class<?> node) {
     private static final BriefcasePreferences APPLICATION_SCOPED =
         new BriefcasePreferences(BriefcasePreferences.class, PreferenceScope.APPLICATION);
   }
+  
+  public static String getBriefCaseProxyType() {
```

Comment:
It also might be cleaner/more efficient to have getProxyProperty return 'null' in the case of no proxy, rather than pass around a model with all empty strings in this case.  See related comments below.